scripts/src/mock_client.ts

```diff
@@ -42,6 +42,20 @@ class SampleAppClient {
 		return res.data.jwt;
 	}
 
+	public async getP2PJWT(data: {
+		offer_id: string;
+		amount: number;
+		sender_title: string;
+		sender_description: string;
+		recipient_id: string;
+		recipient_title: string;
+		recipient_description: string; }) {
+
+		const datastr = Object.keys(data).map(key => `${ key }=${ data[key as keyof typeof data] }`).join("&");
```

Comment:
```
const datastr = Object.keys(data).map(key: keyof typeof data => `${ key }=${ data[key] }`).join("&");
```

Reply:
yeah, that doesn't work, i tried it but the compiler doesn't like it (the keys are string as far as it is concerned, and there's no way to convince it to narrow it down